crates/query-engine/translation/tests/goldenfiles/aggregate_distinct_albums/request.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "../../../../../../static/query.schema.json",
```

Comment:
Generating the schema and putting it in the requests as you have done is a great idea to make tests easier to write, you should do it in a PR before this one and merge it to reduce the amount of noise here.

Reply:
Mixing up functional and non-functional changes like this makes the important details easier to miss.

Reply:
Absolutely fair, will split this into it's own PR

Reply:
Thanks!

Reply:
Done! https://github.com/hasura/ndc-postgres/pull/680